Bar

abbystarnes opened the pull request:
**Changed**

- added styling to bar chart

rodet committed:
First draft of the bar charts

- License and readme created
- First code from the draft we had
- Linters are disabled for now, that will need to be fixed later

rodet committed:
Merge remote-tracking branch 'IBM-Design/master' into bar

seejamescode committed:
Just extra indent

seejamescode committed:
Merge pull request #53 from rodet/bar

First draft of the vanilla bar chart

seejamescode committed:
Live reload server for dev mode

seejamescode committed:
Fixed css link

seejamescode committed:
Merged latest master

seejamescode committed:
Grid animation

rodet committed:
Avoid delivering to dist/

rodet committed:
Merge remote-tracking branch 'origin/line' into IBM-Design/bar

rodet committed:
Merged in the build infrastructure from charts-line

Commit:
css & js styling

Commit:
style w/css and js

seejamescode commented:
Hey @abbystarnes, there are a bunch of `line` files in here. I believe this branch should only reflect `master` branch and that branch does not have a folder for the line chart yet.

Commit:
rewrite bar chart based on line chart

abbystarnes commented:
@seejamescode Thanks for catching that. Not sure how I added the line files, but I've removed them. I rewrote the bar chart based on the line chart so that they'd be in better alignment.

What can I do to pass Travis CI tests?

seejamescode commented:
@abbystarnes Travis looks like it is failing on building the line chart. There is still a `charts-line` folder in your branch, but should not be. See if completely taking out references to the line chart helps!

Commit:
remove line chart folder

abbystarnes committed:
Merge branch 'master' into bar

Commit:
linting

abbystarnes commented:
@seejamescode  I've removed the line folder. Travis CI still seems to be getting stuck on variables that are declared in one file and used in another. (no-unused-vars and no-undef). Is there a work around for this?

rodet commented:
I'm taking a look at the build errors.

rodet committed:
Bar code cleaning (#59)

* Remove unneeded dist folders

* SCSS linter fixes

* Fix the compiling errors

- Remove extraneous file charts-bar.js and its reference in package.json
- Fix linter errors in ivm-charts-bar.js
